commons/signservice-commons/src/main/java/se/idsec/signservice/security/sign/impl/DefaultSignatureValidationResult.java

```diff
@@ -198,6 +208,48 @@ public void setCertificateValidationResult(final CertificateValidationResult cer
     this.certificateValidationResult = certificateValidationResult;
   }
 
+  /** {@inheritDoc} */
+  @Override public String getSignatureAlgorithm() {
+    return signatureAlgorithm;
```

Comment:
@Override on its own line
this.signatureAlgorithm 

Just to be consistent with other code

Reply:
Fixed